sites/public/__tests__/components/applications/StatusItem.test.tsx

```diff
@@ -2,38 +2,117 @@ import React from "react"
 import { render, cleanup } from "@testing-library/react"
 import { t } from "@bloom-housing/ui-components"
 import { StatusItem } from "../../../src/components/account/StatusItem"
+import { ListingsStatusEnum } from "@bloom-housing/shared-helpers/src/types/backend-swagger"
 
 afterEach(cleanup)
 
 describe("<StatusItem>", () => {
-  it("renders without error", () => {
-    const { getByText } = render(
+  it("renders open application without error", () => {
+    const { getByText, queryByText } = render(
       <StatusItem
         applicationDueDate={"March 10th, 2022"}
         applicationURL={"application/1234abcd"}
-        applicationUpdatedAt={"March 8th, 2022"}
         confirmationNumber={"1234abcd"}
         listingName={"Listing Name"}
+        listingStatus={ListingsStatusEnum.active}
         listingURL={"/listing/abcd1234/listing-name"}
       />
     )
 
     expect(getByText("Listing Name")).not.toBeNull()
-    expect(getByText(t("listings.applicationDeadline"), { exact: false })).not.toBeNull()
+    expect(getByText(t("account.openApplications"), { exact: false })).not.toBeNull()
+    expect(getByText(t("account.applicationsClose"), { exact: false })).not.toBeNull()
```

Comment:
Is there a reason these all have to have the `{exact: false}` portion? Looking at the code it looks like the string is a one to one in a span.

nit: `.toBeInTheDocument()` is slightly better than `.not.toBeNull()` https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument

Reply:
Good point, I cleaned that up and switched to toBeInTheDocument()